models/staging/stg_servicenow__change_request.sql

```diff
@@ -69,6 +71,7 @@ final as (
         type as change_request_type,
         unauthorized as is_change_request_unauthorized
     from fields
+    where not _fivetran_deleted
```

Comment:
For all of these let's make the following update. Just to ensure for `null` records that we don't run into any issues.

```suggestion
    where not coalesce(_fivetran_deleted, false)
```

Reply:
thanks! applied to all cases of this